### ISSUE TITLE
BetterPlayer depends on WakeLock ^0.5.6 - cannot upgrade WakieLock to ^0.6.1+1 latest version

### ISSUE THREAD

**Issue by Zelfapp** [jhomlala/betterplayer#898]
Trying to upgrade [WakeLock to ^0.6.1+1 latest version](https://pub.dev/packages/wakelock), but getting the following error running `flutter pub get`.

```
Because better_player 0.0.81 depends on wakelock ^0.5.6 and no versions of better_player match >0.0.81 <0.1.0, better_player ^0.0.81 requires wakelock ^0.5.6.
So, because wakie_up depends on both better_player ^0.0.81 and wakelock ^0.6.1+1, version solving failed.
```

**Comment by shiyiya** [comment]
dependency_overrides:
   wakelock: *

**Comment by jhomlala** [comment]
Fixed in latest release (0.0.82).
